src/library/scala/Option.scala

```diff
@@ -313,6 +442,14 @@ sealed abstract class Option[+A] extends Product with Serializable {
 
   /** Returns a singleton list containing the $option's value
    * if it is nonempty, or the empty list if the $option is empty.
+   *
+   * This is equivalent to:
+   * {{{
+   * option match {
+   *   case Some(x) => List(x)
```

Comment:
style question: do we generally prefer `List(x)` to `x :: Nil`? I assume this has come up in other places

Reply:
There are only a dozen cases of `:: Nil` in the library, but at least twice as many examples of the `List()` constructor.